src/components/Dashboard/Form/Applicants/Applicant.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import FieldWrapper from "../FieldWrapper/FieldWrapper";
+import styles from "./Applicant.scss";
+
+class Applicant extends Component {
+  state = {
+    isVisible: false
+  };
+  handleOnButton = () => {
```

Comment:
Maybe:
`handleOnButton = () => {
this.setState({ isVisible: !this.state.isVisible })
}`

Reply:
yes :) done